python/ml4ir/base/features/feature_config.py

```diff
@@ -327,6 +347,26 @@ def get_feature(self, name: str):
 
         raise KeyError("No feature named {} in FeatureConfig".format(name))
 
+    def feature_exists(self, name: str, trainable=True):
```

Comment:
this function does more than checking if a feature simply exists. Also, some params are missing in the docstring. 
I am wondering if there is value in making it two functions `trainable_feature_exists` and `not_trainable_feature_exists`.

Reply:
this was an easy comment to fix, at least the docstring update.